### ISSUE TITLE
Charge Tracker: Investigate the possibility of tracking start/end at B->C/C->B transitions.

### ISSUE THREAD

**Issue by rtrbt** [Tinkerforge/esp32-firmware#175]
https://www.tinkerunity.org/topic/9013-warp1-mit-206-ende-ladevorgang-f%C3%BCr-ladetracker/#comment-46517

**Comment by Frankstar** [comment]
cant we us `evse/control_pilot_disconnect_update` for that purpose?

**Comment by rtrbt** [comment]
No. Disconnecting the control pilot technically brings the charger into state A, but as we use the disconnect to
a) switch between 1 and 3 phases with the energy manager and
b) wake up sleeping vehicles
the EVSE does not report the state change, so a running charge is not interrupted.
